### PR TITLE
MAT-7002: Add required instruction and asterisk for description.

### PR DESCRIPTION
**Pull request by mcmcphillips**
## MADiE PR

Jira Ticket: [MAT-7002](https://jira.cms.gov/browse/MAT-7002)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [x] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [x] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/components/editMeasure/details/measureMetadata/MeasureMetadata.tsx [hunk 1]
```diff
@@ -119,9 +121,26 @@ export default function MeasureMetadata(props: MeasureMetadataProps) {
...
+              Indicates required field
+            </Typography>
+          ) : (
+            // spacing element to prevent discrepency.
+            <div style={{ height: 15, marginBottom: 6 }} />
+          )}
         </div>
-        <label htmlFor={`measure-${typeLower}`}>{measureMetadataType}</label>
         <TextArea
+          label={measureMetadataType}
```
Comment (adongare) [inline src/components/editMeasure/details/measureMetadata/MeasureMetadata.tsx:142]: 👍

**Review by adongare — APPROVED** [review]
